test/distributed/org/apache/cassandra/fuzz/sai/SingleNodeSAITestBase.java

```diff
@@ -116,6 +116,7 @@ public void basicSaiTest()
                                                          ColumnSpec.regularColumn("v3", ColumnSpec.int64Type)),
                                            List.of(ColumnSpec.staticColumn("s1", ColumnSpec.asciiType(40, 100))),
                                            withAccord ? Optional.of(TransactionalMode.full) : Optional.empty())
+                            .withWriteTimeFromAccord(false) // use the harry timestamp
```

Comment:
spoke with Ariel to confirm we want this behavior for these tests.  I don't know *why* harry fails saying the values don't match what is expected... I assume the writes are out of order but didn't confirm...  According to Ariel `USING TIMESTAMP` could be allowed and the visibility then degrades from strict serializable and is up to the user...

TIL - This is allowed with `full`, only as long as you don't wrap in a `BEGIN TRANSACTION`

Reply:
I wish it was simpler to turn off timestamps in harry; glad this is going to be the case going forward.

Reply:
I wouldn't want to do this in this patch, maybe in v2, but we might want to start prototyping different styles of tests to see what "should" the API look like and what we need.